barcodes_gs1_label/static/src/less/styles.less

```diff
@@ -0,0 +1,364 @@
+//// GLOBAL
```

Comment:
@MiquelRForgeFlow can you convert to .scss?

Reply:
done